mcrouter/ProxyDestination-inl.h

```diff
@@ -161,6 +162,12 @@ void ProxyDestination<Transport>::onReply(
   ++(*stats().results)[static_cast<size_t>(result)];
   destreqCtx.endTime = nowUs();
 
+  if (destreqCtx.requestClass.is(RequestClass::kFailover)) {
```

Comment:
There's also a "normal" request class, for which there can be another counter. But I figured it would be redundant since there are per-result-type counters already.